lib/rubocop/cop/rails/duplicate_association.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop looks for associations that have been defined multiple times in the same file.
+      #
+      # When an association is defined multiple times on a model, ActiveRecord overrides the
```

Comment:
Nice!
This is a nitpick. Maybe "Active Record" seems to be suitable this one.

> The proper names of Rails components have a space in between the words, like "Active Support". `ActiveRecord` is a Ruby module, whereas Active Record is an ORM. 

https://guides.rubyonrails.org/api_documentation_guidelines.html#wording

```suggestion
      # When an association is defined multiple times on a model, Active Record overrides the
```

Reply:
Done!